### PR TITLE
Update episode 01 version control

### PR DESCRIPTION
**Pull request by SarahAlidoost**

### CODE REVIEW

**Review by mkuzak — APPROVED** [review]

File: _episodes/01-version-control.md [hunk 1]
```diff
@@ -12,35 +12,88 @@ objectives:
...
-- "Version control systems help to organize and collaborate on code"
-- "Using a public repository contributes to the reproducibility, reusability and quality of your code"
+- "Version control systems help to organize and collaborate on code."
+- "Using a public repository contributes to the reproducibility, reusability and quality of your code."
 ---
-The [first principle](https://fair-software.eu/recommendations/repository) states:
-> ## Principle 1
-> Use a publicly accessible repository with version control
+
+> ## [Principle 1](https://fair-software.eu/recommendations/repository)
```
Comment (mkuzak) [inline _episodes/01-version-control.md]:
```suggestion
> ## [Recommendation 1](https://fair-software.eu/recommendations/repository)
```

File: _episodes/01-version-control.md [hunk 2]
```diff
@@ -12,35 +12,88 @@ objectives:
...
+>
+> Use a publicly accessible repository with version control.
 {: .callout}
+![principle1]({{ page.root }}/fig/principle1.png)
```
Comment (mkuzak) [inline _episodes/01-version-control.md]: change this to use 'recommendation' instead of 'principle' not to confuse with original FAIR data principles

**Comment by mkuzak** [comment]
Fantastic, thank you @SarahAlidoost . I suggest one change. We should avoid using 'principle', when we talk about fair-software.eu recommendations, not to confuse them with original FAIR data principles

File: _episodes/01-version-control.md (hunk 2 above)
Comment (SarahAlidoost) [reply _episodes/01-version-control.md]: Done!

**Comment by SarahAlidoost** [comment]
> Fantastic, thank you @SarahAlidoost . I suggest one change. We should avoid using 'principle', when we talk about fair-software.eu recommendations, not to confuse them with original FAIR data principles

Thank you @mkuzak . I fixed it here. I will do it for other PR as well.

**Pull request merged**